packages/nitro-protocol/contracts/ETHAssetHolder.sol

```diff
@@ -69,7 +69,7 @@ contract ETHAssetHolder is AssetHolder {
      * @param amount Quantity of wei to be transferred.
      */
     function _transferAsset(address payable destination, uint256 amount) internal override {
-        (bool success, ) = destination.call{value: amount}('');
-        require(success, 'Could not transfer asset');
+        destination.send(amount); // forwards a stipend of only 2300 gas, will not revert on failure
```

Comment:
Does this mean that if the assets are transferred to a contracts with an expensive fallback functions, the gas limit of 2300 will result in the assets being stuck in the asset holder forever?

Reply:
Yes.

Perhaps a better solution would be to rename this function` _transferAssetWithGasStipendAndIgnoreFailures`, and have that used by `_transferAll` and `_claimAll` (which are in some sense "collective" methods called by one participant on behalf of all participants, and therefore the ones that need to be careful about gas siphoning or other griefing problems such as transfer failure), and have a regular `_transferAsset` function which forwards all gas and requires success, and have _that_ used by `_transfer` (which is not a "collective" method).

Reply:
Of course this isn't a really satisfactory solution, since funds could still get locked forever if `transferAll` is called first.

Another idea would be to maintain the requirement for the external call to succeed, and tackle[ _5.2 Recipient can block transferAsset/payouts for all participants (Major)_](https://consensys.net/diligence/audits/private/ljryv8j1b18epn/) in a different way -- namely, by the recently-introduced `transfer` function. In the case of a `_transferAsset` reverting, the participants would need to fall back to each calling `transfer` separately (and spend more gas in the process).

Reply:
Thank for the explanations @geoknee. I think it is good to highlight these issues, but, to be clear, I am not pushing for any immediate action :)